travis/test_server.py

```diff
@@ -302,7 +298,12 @@ def main(argv=None):
             test_loghandler = 'openerp.tools.yaml_import:DEBUG'
     odoo_full = os.environ.get("ODOO_REPO", "odoo/odoo")
     server_path = get_server_path(odoo_full, odoo_version, travis_home)
-    script_name = get_server_script(odoo_version)
+    # Normalize on v10 odoo-bin command
+    try:
+      copy(os.path.join(server_path, 'openerp-server')), 
```

Comment:
Sorry, my https://github.com/OCA/maintainer-quality-tools/issues/398#issuecomment-267404414 was just a  example of the idea. It shouldn't considered real code.